pkg/account/handler.go

```diff
@@ -0,0 +1,202 @@
+package account
+
+import (
+	"container/list"
+	"crypto/rand"
+	"net/http"
+	"regexp"
+
+	"github.com/labstack/echo/v4"
+)
+
+var account_list = list.New()
+
+// type AcountType string
+
+type AccountHandler struct {
+}
+
+// @Summary create and account based on email(as id), type, name and password
+// @Description will check primarykey other, then add to account_list if possible
+// @Tags Account
+// @Produce json
+// @Param email formData string true "user e-mail"
+// @Param type formData string true "user type"
+// @Param name formData string true "user name"
+// @Param paswd formData string true "user password"
+// @Success 200 {string} string "{"msg": "Successfully created"}"
+// @Failure 400 {string} string "{"error": "Invali E-mail Address"}"
+// @Router /case/PastHistory/{id} [PATCH]
```

Comment:
And use POST rather than PATCH to create, PATCH is not widely used. Use PUT for modification.

Reply:
And url should be all lower case.
And the route path looks suspicous, what is `/case/PastHistory/{id}`?